app/Http/Controllers/Api/Manager/AttendanceController.php

```diff
@@ -35,49 +35,32 @@ class AttendanceController extends Controller
      */
     public function store(StoreRequest $request): JsonResponse
     {
-        $managerId = $request->user()->id;
-
-        /** @var Instructor $manager */
-        $manager = Instructor::with('managings')->findOrFail($managerId);
-
-        $instructorIds = $manager->managings->pluck('id')->toArray();
-        $instructorIds[] = $manager->id;
-
-        $courseIds = Course::with('instructor')
-            ->whereIn('instructor_id', $instructorIds)
-            ->pluck('id')
-            ->toArray();
-
-        /** @var Course $course */
-        $course = Course::find($request->course_id);
+        /** @var Attendance $attendance */
+        $course = Course::findOrFail($request->course_id);
 
-        if (! in_array($course->id, $courseIds, true)) {
-            // 自分もしくは配下の講師の講座でない場合はエラーを返す
-            throw new AuthorizationException('Forbidden.');
-        }
+        // Policyによる認可チェック
+        $this->authorize('create', $course);
```

Comment:
明示的に`AttendancePolicy::create`として呼び出すために次のように修正をお願いします。

```suggestion
        // Policyによる認可チェック
        $this->authorize('create', [Attendance::class, $course]);
```